virtual_motor.c

```diff
@@ -214,7 +214,7 @@ static void connect_virtual_motor(float ml , float J, float Vbus){
 																							GET_GATE_DRIVER_SUPPLY_VOLTAGE();
 		}
 #endif
-		virtual_motor.phi = mcpwm_foc_get_phase() * M_PI / 180.0;
+		virtual_motor.phi = DEG2RAD_f(mcpwm_foc_get_phase());
```

Comment:
One reason to write macros with CAPITALS, is to be aware that the argument could be evaulated twice. That would be bad if the argument evaluation is slow or have side-effects. Don't know if we should care about that here and use a temporary variable, or if we should use inline functions instead of macros. It is fine with me the current way though, just saying the history why macros are written CAPITALIZED.

See for example `UTILS_IS_NAN(x)` and `UTILS_NAN_ZERO(x)` that evaluates their argument twice.

Reply:
You mean `DEG2RAD_f(DEG2RAD_f(x))` as a risk?

This makes somewhat of an argument for returning to pure multiplication. There are no sharp edges to `#define DEG2RAD_f (180.0f/M_PI)`.

Reply:
I still prefer the macros with arguments over multiplications. It is easier to see how to use them and it is less likely to make mistakes with parentheses when using them compared to multiplications. These macros are not evaluating arguments twice either, so that is a general statement about macros that sometimes is true but not here.

Reply:
I'm not one to let the perfect get in the way of the good. If you're happy with it and it tests out okay, I think it's ready for merge.